### PR TITLE
update the function to get the image tag

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### What problem does this PR solve? 
The original way to get the image tag did not consider the "127.0.0.1:30002/pingcap/tidb-operator:v3.1.0" case.
### What is changed and how does it work?
Update the function to get the image tag.
### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
   - Backup and Restore with v3.1.0-rc
   - Backup and Restore with nightly

Code changes

 - Has Go code change

Related changes

 - Need to cherry-pick to the release branch

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: pkg/backup/util/util.go [hunk 1]
```diff
@@ -410,3 +410,13 @@ func ValidateRestore(restore *v1alpha1.Restore) error {
...
+
+// GetImageTag get the image tag
+func GetImageTag(image string) string {
+	var tag string
+	colonIdx := strings.LastIndexByte(image, ':')
+	if colonIdx >= 0 {
+		tag = image[colonIdx+1:]
+	}
+	return tag
+}
```
Comment (Yisaer) [inline pkg/backup/util/util.go:422]: add a unit test.

File: pkg/backup/util/util.go [hunk 2]
```diff
@@ -410,3 +410,13 @@ func ValidateRestore(restore *v1alpha1.Restore) error {
 	}
 	return nil
 }
+
+// GetImageTag get the image tag
```
Comment (cofyc) [inline pkg/backup/util/util.go:414]: `gets` or `returns`

File: pkg/backup/util/util.go [hunk 3]
```diff
@@ -410,3 +410,13 @@ func ValidateRestore(restore *v1alpha1.Restore) error {
...
+func GetImageTag(image string) string {
```
Comment (cofyc) [inline pkg/backup/util/util.go:415]: update it to return `image` and `tag`, e.g. `func PaseImageTag(image string) (string, string)`.
in your case, discard the first result, e.g. `_, tikvVersion := PaseImageTag(...)`.
make it generic, then we can use it in more places.

**Comment by DanielZhangQD** [comment]
@Yisaer @cofyc comments addressed, PTAL.

File: pkg/backup/util/util.go (hunk 1 above)
Comment (DanielZhangQD) [reply pkg/backup/util/util.go]: Done.

File: pkg/backup/util/util.go (hunk 2 above)
Comment (DanielZhangQD) [reply pkg/backup/util/util.go]: Done.

File: pkg/backup/util/util.go (hunk 3 above)
Comment (DanielZhangQD) [reply pkg/backup/util/util.go]: Done.

**Pull request merged**